### ISSUE TITLE
Too many reds for mālā?

### ISSUE THREAD

**Issue by gasyoun** [shreevatsa/sanskrit#118]
I have an issue to report about the results for the following input:
```
वने रणे शत्रुजलाग्निमध्ये
	महार्णवे पर्वतमस्तके वा ।
सुप्तं प्रमत्तं विषमस्थितं वा
	रक्षन्ति पुण्यानि पुराकृतानि ॥ ९९ ॥

```
![mala](https://user-images.githubusercontent.com/80761/92147023-39d8d880-ee23-11ea-8cc7-fbf7680c2bdd.PNG)

Still ok?

**Comment by shreevatsa** [comment]
Yes this is clearly a bug. Seems to be happening because of conflict between:

https://github.com/shreevatsa/sanskrit/blob/6748493f7bf0056e7413e7d69a9c33df47ecd1ec/data/mishra.json#L595

and

https://github.com/shreevatsa/sanskrit/blob/6748493f7bf0056e7413e7d69a9c33df47ecd1ec/data/vrttaratnakara.json#L94

(The verse actually perfectly matches the first one, but alignment is incorrectly using the second one.)
